pnpm-lock.yaml

```diff
@@ -1,23 +1,7 @@
 lockfileVersion: '6.0'
 
 overrides:
-  axios: ^1.6.0
-  dns-packet: ^1.3.2
-  follow-redirects: ^1.15.4
-  glob-parent: ^5.1.2
```

Comment:
Removing some of these will reintroduce old dependabot/CG issues

Reply:
I believe string-decoder and semver are the key culprits, but I'll double check and get back to you

Reply:
minimist version can't be below 0.2.4

Reply:
Updated `minimist` to use latest version.

Reply:
This shouldn't be an issue anymore as `resolutions` field is updated to `pnpm.overrides`